.github/workflows/k8s-e2e.yml

```diff
@@ -91,10 +89,8 @@ jobs:
 
           [plugins."io.containerd.grpc.v1.cri".registry.mirrors."${registry_ip}:5000"]
               endpoint = ["http://${registry_ip}:5000"]
-          [plugins."io.containerd.grpc.v1.cri".registry.configs."${registry_ip}:5000".auth]
-            username = "${{ env.DOCKER_USER }}"
-            password = "${{ env.DOCKER_PASSWORD }}"
```

Comment:
Is this not required?
I see the CI passed, but it seems both `cri` and `kubeconf` are useing the same node to run test. 
You added `image-service-endpoint` to kubelet, but not delete it when running `kubeconfg` test.

Reply:
From [using-jobs-in-a-workflow](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#overview) and [using-a-github-hosted-runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#using-a-github-hosted-runner):

> Each job runs in a runner environment specified by runs-on.
> When the job begins, GitHub automatically provisions a new VM for that job.

I think each job will run in a standalone runner environment(VM) and don't share any data 🤔

> Is this not required?

Under kubeconf case, kubelet can get the image secret from `imagePullSecrets` as well and pass it to containerd now.

Reply:
I‘ve tried to run a single kubeconf case: https://github.com/darfux/nydus-snapshotter/actions/runs/3701944536/jobs/6271709388 and it seems still OK. @liubin FYI😀